community/modules/scripts/ramble-setup/main.tf

```diff
@@ -36,9 +36,10 @@ locals {
       install_dir           = var.install_dir
       git_url               = var.ramble_url
       git_ref               = var.ramble_ref
-      chown_owner           = var.chown_owner == null ? "" : var.chown_owner
-      chgrp_group           = var.chgrp_group == null ? "" : var.chgrp_group
       chmod_mode            = var.chmod_mode == null ? "" : var.chmod_mode
+      system_user_name      = var.system_user_name
+      system_user_uid       = var.system_user_uid == null ? "" : var.system_user_uid
```

Comment:
Please set `nullable = false` and `default = ""` for var.system_user_uid and var.system_user_gid. You can change the types to string and terraform automatically converts numbers to strings.

https://developer.hashicorp.com/terraform/language/expressions/types#type-conversion

In any case, if you don't want null to be allowed, then do not allow it to be null with `nullable=false`.